tests/tests_pytorch/utilities/test_compile.py

```diff
@@ -57,13 +56,6 @@ def test_trainer_compiled_model(tmp_path, monkeypatch):
     trainer.fit(model)
     assert trainer.model._compiler_ctx is None
 
-    # some strategies do not support it
-    compiled_model = torch.compile(model)
-    mock_cuda_count(monkeypatch, 1)
-    trainer = Trainer(strategy="dp", accelerator="cuda", **trainer_kwargs)
```

Comment:
This should be kept but using a different strategy that is not supported with `torch.compile`